README.md

```diff
@@ -1 +1,131 @@
-# Leaderboard
\ No newline at end of file
+<a name="LeaderBoard"></a>
+<div align="center">
+  <br/>
+  <h3><b>LeaderBoard</b></h3>
+</div>
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[To_Do_List] \_To\_Do\_List]
+  - [🛠 Built With Html, css, javascript](#-built-with-html-css-javascript)
+    - [Tech Stack HTML5, CSS3 JAVASCRIPT](#tech-stack-html5-css3-javascript)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Author(s) ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+  - [📝 License](#-license)
+
+# 📖 [LeaderBoard] <a name="LeaderBoard">LeaderBoard</a>
+
+A solo programming project that highlights the use of Webpack to bundle JavaScript, ES6 syntax, API, callbacks and promise
+
+## 🛠 Built With <a name="built-with">Html, css, javascript</a>
+
+### Tech Stack <a name="tech-stack">HTML5, CSS3 JAVASCRIPT</a>
+
+HTML5 and css3 which are the latest versions and JavaScript were used in this project.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+-  leaderboard built with HTML CSS and javascript 
+-  Dynamically loading with javascript.
+-  Apply module bundler (Webpack).
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
+
```

Comment:
- Please, since you've not deployed the app, kindly remove this live demo section from the readme. should in case you want to add a deployment later, kindly state that you'd do that soon to keep your readme more professional.